lab-patrick/test/fs-module-test.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const expect = require('chai').expect;
+const fs = require('fs');
+
+
+describe('fs-module', function(){
+  describe('#readFile', function(){
```

Comment:
I see what youre trying to do but its easier if you use the existing for loop that reads the files and using that as your test, then you can just read and expect or use the data that has already been written